app/(main)/login/page.test.tsx

```diff
@@ -62,7 +76,7 @@ describe('Login', () => {
     fireEvent.change(passwordInput, { target: { value: 'password123' } });
     fireEvent.click(continueButton);
 
-    await waitFor(() => {
+    waitFor(() => {
       expect(mockLogin).toHaveBeenCalledWith({
```

Comment:
Typically when using `waitFor(() => )` it's used with an 'async await'. Any thoughts if we should add in `async await`?

Reply:
@shashilo since you wrote this, can you explain?

Reply:
I did not make these changes. I made them in the register page. Mai is correct. async/await is needed.

Reply:
> I did not make these changes. I made them in the register page. Mai is correct. async/await is needed.

My apologies, I copied what you wrote in the Register page for this test, thanks for the clarification. I'll fix this.